src/SearchParameters/index.js

```diff
@@ -197,7 +197,10 @@ function SearchParameters(newParameters) {
 
   var self = this;
   Object.keys(params).forEach(function(paramName) {
-    if (SearchParameters.PARAMETERS.indexOf(paramName) === -1) {
+    var isKeyUnknown = SearchParameters.PARAMETERS.indexOf(paramName) === -1;
```

Comment:
Reversing this expression would make it easier to understand.

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/703/commits/6e705246a09365d3dccd480bc5773f31e24ffb0a